modules/oadp-ceph-cephfs-back-up.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+ifeval::["{context}" == "cephfs"]
+:cephfs:
+endif::[]
+ifeval::["{context}" == "split"]
+:split:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="oadp-ceph-cephfs-back-up_{context}"]
+
+ifdef::cephfs[]
+= Backing up data using OADP 1.2 Data Mover and CephFS storage
+You can use OpenShift API for Data Protection (OADP) 1.2 Data Mover to back up data using CephFS storage by enabling the shallow copy feature of CephFS storage.
+endif::[]
+
+ifdef::split[]
+= Backing up data using OADP 1.2 Data Mover and split volumes
+You can use OpenShift API for Data Protection (OADP) 1.2 Data Mover to back up data in an environment that has split volumes.
+endif::[]
```

Comment:
idr the rules about this, but I see ones in the repo seem to have the condition spelled out in the endif for the body text like this
```suggestion
ifdef::cephfs[]
= Backing up data using OADP 1.2 Data Mover and CephFS storage
You can use OpenShift API for Data Protection (OADP) 1.2 Data Mover to back up data using CephFS storage by enabling the shallow copy feature of CephFS storage.
endif::[]

ifdef::split[]
= Backing up data using OADP 1.2 Data Mover and split volumes
You can use OpenShift API for Data Protection (OADP) 1.2 Data Mover to back up data in an environment that has split volumes.
endif::split[]
```